calculation/gmhazard_calc/gmhazard_calc/directivity/scripts/mc_nhyp_runs.py

```diff
@@ -68,68 +118,66 @@ def perform_mp_directivity(
         np.exp(total_fd_array),
     )
     np.save(
-        f"{output_dir}/{fault_name}_{nhypo}_fd_mc.npy",
-        np.exp(total_fd),
+        f"{output_dir}/{fault_name}_{nhypo}_fd_mc.npy", np.exp(total_fd),
     )
     np.save(
-        f"{output_dir}/{fault_name}_{nhypo}_fd_average.npy",
-        np.exp(fdi_average),
+        f"{output_dir}/{fault_name}_{nhypo}_fd_average.npy", np.exp(fdi_average),
     )
 
 
 def parse_args():
-    nhm_dict, faults, im, grid_space, nhyps = common.default_variables()
+    # nhm_dict, faults, im, grid_space, nhyps = common.default_variables()
+
+    im = IM(IMType.pSA, period=3.0)
+    nhm_dict = nhm.load_nhm("/mnt/data/work/seistech/sources/18p6/NZ_FLTmodel_2010.txt")
+    faults = ["AlpineK2T", "AlfMakuri", "Wairau", "ArielNorth", "Swedge1", "Ashley"]
+    nhyps = [3, 4, 5, 6, 7, 8, 9, 10, 15, 20, 25, 30]
+    grid_space = 100
```

Comment:
Would it be better with a prefix? something like `default_`.
Until I read `default=faults`, I was about to leave comments asking whether we need these.

Reply:
This was just a hack I had added (to get around another hack lol), gone now.